web/src/portico/landing-page.js

```diff
@@ -164,17 +164,6 @@ $(() => {
             // Set the correct values for span and colspan
             $(".features-col-group").attr("span", plans_columns_count);
```

Comment:
```
        // Not all pages that show plans include the comparison
        // table, but when it's present, make sure to align the
        // comparison table with the current active plans tab
        if ($comparison_table.length > 0) {
```

We don't need this if condition since we are already doing a check for `window.location.pathname.endsWith("/plans/")` above.

Reply:
Ah, that's true.